scripts/pre_push_hook.py

```diff
@@ -122,10 +146,10 @@ def _collect_files_being_pushed(ref_list, remote):
         ref_list: list of references to parse (provided by git in stdin)
         remote: the remote being pushed to
     Returns:
```

Comment:
Couple of questions:

(1) Under what circumstances will there be more than one branch under consideration?
(2) In the circumstances described in (1), why don't we just lint the branch being pushed, instead of linting all branches?